source/linux/Foundational_Components/Kernel/Kernel_Drivers/QSPI.rst

```diff
@@ -210,50 +182,50 @@ Start the Linux Kernel Configuration tool:
 
 .. code-block:: console
 
-        $ make menuconfig ARCH=<architecture_name>
+    $ make menuconfig ARCH=<architecture_name>
```

Comment:
Over 50% of this file is using the proper 3 space indentation. Please standardize these code-blocks if you are going to adjust the whitespace.

This goes for every modified code-block in this PR.